app/views/manage_competencies/history/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<% title t('.page_title') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl">
+      <%= t '.heading', user_name: @caseworker.name_or_email %>
+    </h1>
+  </div>
+
+  <div class="govuk-grid-column-full">
+    <h2 class="govuk-heading-m">
+      History
+    </h2>
+    <% if @caseworker.competency_history.items.present? %>
+      <table class="govuk-table app-dashboard-table">
+        <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">
+            <%= thead_text(:when) %>
+          </th>
+          <th scope="col" class="govuk-table__header govuk-!-width-one-half">
+            <%= thead_text(:what) %>
+          </th>
+          <th scope="col" class="govuk-table__header">
+            <%= thead_text(:supervisor) %>
+          </th>
+        </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        <% @caseworker.competency_history.items.each do |event| %>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">
+              <%= l event.timestamp, format: :datetime %>
+            </td>
+            <td class="govuk-table__cell">
+              <% if event.data[:competencies].empty? %>
+                <%= t '.competencies_set', competencies_text: 'no competencies' %>
+              <% else %>
+                <%= t '.competencies_set', competencies_text: event.data[:competencies].map(&:humanize).join(', ') %>
```

Comment:
Probably could do similar here though not a big deal either way tbh

Reply:
Probably best to be consistent with the pattern we've just established. I need to get into the habit of using presenters 😄